tap_github/__init__.py

```diff
@@ -1051,11 +1051,15 @@ def should_skip_repo(
     include_disabled: bool,
 ):
     name = repository.get('name')
+    size = repository.get('size')
     is_archived = repository.get('archived', False)
     is_disabled = repository.get('disabled', False)
     match_excludes = next(filter(lambda exclude: fnmatch.fnmatch(name, exclude), excludes), None)
     match_includes = next(filter(lambda include: fnmatch.fnmatch(name, include), includes), None)
 
+    if size <= 0:
```

Comment:
After my querying, I didn't have confidence that the size parameter for repos is actually accurate enough to be useful for this - are you sure that the repos which have no commits all have size=0?

Reply:
yes, that's what the github api returns. 
I did the test for multiple repos